README.md

```diff
@@ -57,4 +57,7 @@ For details on cross-compilation, check out the [GitHub Actions file](.github/wo
 
 ## License
 
-`jq` is released under the [MIT License](COPYING).
+`jq` is released under the [MIT License](COPYING). The jq's documentation is
```

Comment:
`The jq's documentation` => <code>&#96;jq&#96;'s documentation</code>?

Reply:
Agreed.  Other than that LGTM!

Reply:
Thanks for feedback, PR updated

Reply:
@thalman You didn't remove `The`

Reply:
Ahh sorry, I missed that and fixed just formatting. Removed.